image/cli/mascli/templates/pipelinerun.yaml

```diff
@@ -19,6 +19,10 @@ spec:
       value: '$ARTIFACTORY_USERNAME'
     - name: artifactory_apikey
       value: '$ARTIFACTORY_APIKEY'
+    - name: dev_mode_airgap
+      value: '$DEV_MODE_AIRGAP'
+    - name: catalog_digest
```

Comment:
Can we use `mas_catalog_digest` to align with the namine conventions for other catalog variables? e.g. `mas_catalog_version`